lib/cli_interface.rb

```diff
@@ -112,28 +148,36 @@ def get_from_file(text)
   end
 
   def display_line(row)
+    content = " "
+
     row.each_with_index do |e, index|
       if index == row.size - 1
-        write(" #{e} \n")
+        write("#{e}".center(margin(row), content)  + "\n")
       else
-        write(" #{e} |")
+        write("#{e}".center(margin(row), content)  + "|")
       end
     end
   end
 
-  def display_lang_files
-    menu = "#{get_from_file('menu_lang')}\n"
-    language = ""
+  def display_interline(row)
+    write( "-" * (row.size + margin(row) * row.size - 1) + "\n")
+  end
+
+  def margin(row)
+      (row.size ** 2).to_s.size + 2
+  end
+
+  def get_menu_lang
+    menu_lang = "#{get_from_file('menu_lang')}\n"
+    languages = ""
 
     Dir.entries(@source).each do |item, i|
       if item.end_with?("#{@FILE_EXTENTION}")
-        @count_lang += 1
-        language << " #{@count_lang} - #{item.chomp("#{@FILE_EXTENTION}").capitalize}\n"
+        languages  << "#{item.chomp("#{@FILE_EXTENTION}").capitalize}\n"
```

Comment:
can this be extracted into a method which explains what it's doing.
In fact can this whole Dir block also be extracted into a method that explains what it's doing?